agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/BeforeAgentInstaller.java

```diff
@@ -187,6 +196,44 @@ public void run() {
                 }
             }
         });
+
+        pollJsonConfigEveryMinute();
+    }
+
+    private static volatile Long lastModifiedTime;
+    private static void pollJsonConfigEveryMinute() {
+        Executors.newSingleThreadScheduledExecutor().scheduleWithFixedDelay(new Runnable() {
```

Comment:
need to make this a daemon thread so it doesn't prevent JVM shutdown

also, let's give the thread a name, for when it shows up in thread dump